stablehlo/transforms/Passes.td

```diff
@@ -96,3 +96,13 @@ def VhloToVersionPass : Pass<"vhlo-to-version"> {
            "The target version. Must be a version of the form #.#.# or 'current'.">,
   ];
 }
+
+def LegalizeChlo :
```

Comment:
For consistency:

`ChloLegalizeToStablehloPass` and `chlo-legalize-to-stablehlo`